MapleServer2/Constants/Skills/SkillTreeOrdered.cs

```diff
@@ -0,0 +1,271 @@
+            11000291
+        };
+        private static readonly List<int> SoulBinder = new List<int>
+        {
+            11100111,11100112,11100271,11100061,11100062,11100221,11100011,11100171,11100121,11100281,
+            11100071,11100231,11100072,11100232,11100073,11100074,11100021,11100181,11100075,11100022,
+            11100023,11100024,11100131,11100025,11100184,11100132,11100292,11100133,11100081,11100134,
+            20000001,11100241,11100135,11100082,01200001,11100242,11100083,01200002,11100084,11100191,
+            01200003,11100244,11100085,11100192,01200004,11100193,01200005,11100141,11100089,11100091,
+            20000011,01200011,11100041,01200012,11100201,01200013,11100202,01200014,01200015,11100151,
+            11100311,11100153,11100101,11100154,11100261,11100102,11100155,11100103,11100315,11100156,
+            11100051,11100104,11100105,11100031,11100321,11100204,11100187,11100272,11100001,11100291,
+            11100002,11100173,11100243,11100211,11100262,11100161,11100301,11100063,11100251
+        };
+        private static readonly List<int> GameMaster = new List<int>
+        {
+            0
+        };
+        #endregion
+
+        #region Default skills learned.
+        private static readonly List<int> KnightDefault = new List<int>
+        {
+            10100061,10100091
+        };
+        private static readonly List<int> BerserkerDefault = new List<int>
+        {
+            10200031,10200091
+        };
+        private static readonly List<int> WizardDefault = new List<int>
+        {
+            10300061,10300081
+        };
+        private static readonly List<int> PriestDefault = new List<int>
+        {
+            10400111,10400151
+        };
+        private static readonly List<int> ArcherDefault = new List<int>
+        {
+            10500111,10500151
+        };
+        private static readonly List<int> HeavyGunnerDefault = new List<int>
+        {
+            10600071,10600081
+        };
+        private static readonly List<int> ThiefDefault = new List<int>
+        {
+            10700041,10700042,10700061
+        };
+        private static readonly List<int> AssassinDefault = new List<int>
+        {
+            10800071,10800081
+        };
+        private static readonly List<int> RunebladeDefault = new List<int>
+        {
+            10900041,10900051
+        };
+        private static readonly List<int> StrikerDefault = new List<int>
+        {
+            11000051,11000053,11000055,11000081
+        };
+        private static readonly List<int> SoulBinderDefault = new List<int>
+        {
+            11100051,11100061
+        };
+        #endregion
+        /// <summary>
+        /// Get the specific ordered skill list of each Job. Requiered for SkillBookTree
+        /// </summary>
+        public static List<int> GetListOrdered(Job job)
+        {
+            switch (job)
+            {
+                case Job.Knight:
+                    return Knight;
```

Comment:
If you turn the above code into a Dictionary, where the key is the Job and the values is the list of skills, you can eliminate the need for this method and clean up the above code